src/PrestaShopBundle/Routing/Converter/LegacyRoute.php

```diff
@@ -53,6 +53,21 @@ class LegacyRoute
      */
     private $controllersActions;
 
+    /**
+     * LegacyRoute constructor.
```

Comment:
Not blocking for merge: this line does not provide value

Reply:
Fixed